strategies/affected/pom.xml

```diff
@@ -32,7 +32,6 @@
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
-      <scope>test</scope>
```

Comment:
why is the scope removed?

Reply:
Because it is inherited

Reply:
that you can say about the rest of the test dependencies... I would prefer to have everything specified there so I don't have to go to the parent pom checking what is specified there...

Reply:
Originally there were not there the scope. Then I added for testing one thing but I committed by error, and then @bartoszmajsak (And I agree with him) asked me to remove again.

Reply:
originally when? looking at the state before your commit, it was there ;-) https://github.com/arquillian/smart-testing/blob/1498d4051942f3b8725712bcd5fbe547cbd2e82b/strategies/affected/pom.xml#L35

Reply:
check other pom.xmls files and check that it was not present

Reply:
Then, from my point of view, it should be there. Otherwise, you can remove the scope from all other test dependencies

Reply:
We had this kind of conversation already and I do remember that we agreed on having it there, otherwise, it brings a lot of confusions...

Reply:
Something to discuss on Cabal I guess to know how to proceed in general.

Reply:
Yes I remember this but since I saw that test scope was not in any of the modules, then I decided to remove as well to follow the same convention. In fact if you notice the only one that it had before was `affected`

Reply:
as for the state before your commit: 
`find ./ -name "pom.xml" | xargs grep "artifactId>junit<" -A 1`
```xml
./functional-tests/test-bed/pom.xml:      <artifactId>junit</artifactId>
./functional-tests/test-bed/pom.xml-      <scope>compile</scope>
--
./functional-tests/git-rules/pom.xml:      <artifactId>junit</artifactId>
./functional-tests/git-rules/pom.xml-      <scope>compile</scope>
--
./strategies/changed/pom.xml:      <artifactId>junit</artifactId>
./strategies/changed/pom.xml-      <scope>test</scope>
--
./strategies/failed/pom.xml:      <artifactId>junit</artifactId>
./strategies/failed/pom.xml-      <scope>test</scope>
--
./strategies/affected/pom.xml:      <artifactId>junit</artifactId>
./strategies/affected/pom.xml-      <scope>test</scope>
--
./surefire-provider/pom.xml:      <artifactId>junit</artifactId>
./surefire-provider/pom.xml-      <scope>test</scope>
--
./junit-test-result-parser/pom.xml:      <artifactId>junit</artifactId>
./junit-test-result-parser/pom.xml-    </dependency>
--
./mvn-extension/pom.xml:      <artifactId>junit</artifactId>
./mvn-extension/pom.xml-    </dependency>
--
./core/pom.xml:      <artifactId>junit</artifactId>
./core/pom.xml-    </dependency>
```
so, I see more cases where the scope was specified... still, if you wanted to keep it unified, you should have removed it from all other test dependencies, which means everyting that is in the parent pom file: https://github.com/arquillian/smart-testing/blob/1498d4051942f3b8725712bcd5fbe547cbd2e82b/pom.xml#L85-L114...